### PR TITLE
httpie: update to 3.2.3

### PR DESCRIPTION
**Pull request by stdmnpkg**
Topic Description
-----------------

- httpie: update to 3.2.3
    Co-authored-by: (@stdmnpkg)

Package(s) Affected
-------------------

- httpie: 3.2.3

Security Update?
----------------

No

Build Order
-----------

```
#buildit httpie
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] Architecture-independent `noarch`

### CODE REVIEW

**Comment by stdmnpkg** [comment]
fix [CVE-2023-48052](https://nvd.nist.gov/vuln/detail/CVE-2023-48052)

> Missing SSL certificate validation in HTTPie v3.2.2 allows attackers to eavesdrop on communications between the host and server via a man-in-the-middle attack.

**Comment by stdmnpkg** [comment]
works, or more exactly, as broken as previous version on my loongarch64 machine.

additional dependency required, at least require `rich`, also require `pygments` 2.12 which we are on 2.10, ssl cert verification always failed

**Comment by MingcongBai** [comment]
> works, or more exactly, as broken as previous version on my loongarch64 machine.
> 
> additional dependency required, at least require `rich`, also require `pygments` 2.12 which we are on 2.10, ssl cert verification always failed

Please address at least the dependency issues in this topic.

**Comment by stdmnpkg** [comment]
works on my x86-64 machine

image

File: app-web/httpie/autobuild/defines [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PKGNAME=httpie
 PKGSEC=utils
-PKGDEP="defusedxml multidict pygments pysocks requests toolbelt"
+PKGDEP="rich defusedxml multidict pygments pysocks requests toolbelt rich"
```
Comment (KexyBiscuit) [inline app-web/httpie/autobuild/defines]: Redundant dependencies.

**Review by KexyBiscuit — CHANGES_REQUESTED** [review]

**Review by KexyBiscuit — APPROVED** [review]
LGTM.

**Pull request merged**